features/admin/orphan_lists.feature

```diff
@@ -158,7 +158,7 @@ Feature:
     Then I click the "Upload" button
     Then I should see "محمد"
     And I should see "John Doe"
-    And I should see "January 02, 2012"
+    And I should see "2012-01-02"
```

Comment:
This is in app/views/admin/pending_orphan_lists/valid_list.html.arb. It didn't receive the date formatter and uses simply `column :date_of_birth`. Best guess is that default date formatting changed somewhere after gems upgrade.